hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestArchivedCommitsCommand.java

```diff
@@ -0,0 +1,195 @@
+    cleanupDFS();
+  }
+
+  /**
+   * Test for command: show archived commit stats.
+   */
+  @Test
+  public void testShowArchivedCommits() {
+    CommandResult cr = getShell().executeCommand("show archived commit stats");
+    assertTrue(cr.isSuccess());
+
+    TableHeader header = new TableHeader().addTableHeaderField("action").addTableHeaderField("instant")
+        .addTableHeaderField("partition").addTableHeaderField("file_id").addTableHeaderField("prev_instant")
+        .addTableHeaderField("num_writes").addTableHeaderField("num_inserts").addTableHeaderField("num_deletes")
+        .addTableHeaderField("num_update_writes").addTableHeaderField("total_log_files")
+        .addTableHeaderField("total_log_blocks").addTableHeaderField("total_corrupt_log_blocks")
+        .addTableHeaderField("total_rollback_blocks").addTableHeaderField("total_log_records")
+        .addTableHeaderField("total_updated_records_compacted").addTableHeaderField("total_write_bytes")
+        .addTableHeaderField("total_write_errors");
+
+    // Generate expected data
+    final List<Comparable[]> rows = new ArrayList<>();
+    for (int i = 100; i < 104; i++) {
+      String instant = String.valueOf(i);
+      for (int j = 0; j < 3; j++) {
+        Comparable[] defaultComp = new Comparable[]{"commit", instant,
+            HoodieTestCommandDataGenerator.DEFAULT_SECOND_PARTITION_PATH,
+            HoodieTestCommandDataGenerator.DEFAULT_FILEID,
+            HoodieTestCommandDataGenerator.DEFAULT_PRE_COMMIT,
+            HoodieTestCommandDataGenerator.DEFAULT_NUM_WRITES,
+            HoodieTestCommandDataGenerator.DEFAULT_OTHER_VALUE,
+            HoodieTestCommandDataGenerator.DEFAULT_OTHER_VALUE,
+            HoodieTestCommandDataGenerator.DEFAULT_NUM_UPDATE_WRITES,
+            HoodieTestCommandDataGenerator.DEFAULT_NULL_VALUE,
+            HoodieTestCommandDataGenerator.DEFAULT_TOTAL_LOG_BLOCKS,
+            HoodieTestCommandDataGenerator.DEFAULT_OTHER_VALUE,
+            HoodieTestCommandDataGenerator.DEFAULT_OTHER_VALUE,
+            HoodieTestCommandDataGenerator.DEFAULT_TOTAL_LOG_RECORDS,
+            HoodieTestCommandDataGenerator.DEFAULT_OTHER_VALUE,
+            HoodieTestCommandDataGenerator.DEFAULT_TOTAL_WRITE_BYTES,
+            HoodieTestCommandDataGenerator.DEFAULT_OTHER_VALUE};
+        rows.add(defaultComp.clone());
+        defaultComp[2] = HoodieTestCommandDataGenerator.DEFAULT_FIRST_PARTITION_PATH;
+        rows.add(defaultComp);
+      }
+    }
+
+    String expectedResult = HoodiePrintHelper.print(
+        header, new HashMap<>(), "", false, -1, false, rows);
+    assertEquals(expectedResult, cr.getResult().toString());
+  }
+
+  /**
+   * Test for command: show archived commits.
+   */
+  @Test
+  public void testShowCommits() throws IOException {
+    CommandResult cr = getShell().executeCommand("show archived commits");
+    assertTrue(cr.isSuccess());
+    System.out.println(cr.getResult().toString());
```

Comment:
Why we should use `System.out.println`? Do we need to get information from STDOUT?

Reply:
> Why we should use `System.out.println`? Do we need to get information from STDOUT?

Removed it.